packages/desktop/src/services/FileDownloadServiceElectron.ts

```diff
@@ -197,129 +178,22 @@ export default class FileDownloadServiceElectron implements FileDownloadService
-            postData,
-            requestOptions,
-            url,
-            writeStreamOptions: { flags: "w" }, // The file is created (if it does not exist) or truncated (if it exists).
-        });
-    }
-
-    public async promptForDownloadDirectory(): Promise<string> {
-        const title = "Select download directory";
-
-        // Continuously try to set a valid directory location until the user cancels
-        while (true) {
-            const defaultDownloadDirectory = await this.getDefaultDownloadDirectory();
-            const directoryPath = await this.promptUserWithDialog({
-                title,
-                properties: ["openDirectory"],
-                defaultPath: defaultDownloadDirectory,
-            });
-
-            if (directoryPath === FileDownloadCancellationToken) {
-                return FileDownloadCancellationToken;
-            }
-
-            const isDirectory = await FileDownloadServiceElectron.isDirectory(directoryPath);
-            const isWriteable =
-                isDirectory && (await FileDownloadServiceElectron.isWriteable(directoryPath));
-
-            // If the directory has passed validation, return
-            if (isDirectory && isWriteable) {
-                return directoryPath;
-            }
-
-            // Otherwise if the directory failed validation, alert
-            // user to error with executable location
-            let errorMessage = `Whoops! ${directoryPath} is not verifiably a directory on your computer.`;
-            if (isDirectory && !isWriteable) {
-                errorMessage += ` Directory does not appear to be writeable by the current user.`;
-            }
-            await this.notificationService.showError(title, errorMessage);
-        }
-    }
-
-    public getDefaultDownloadDirectory(): Promise<string> {
-        return ipcRenderer.invoke(FileDownloadServiceElectron.GET_DOWNLOADS_DIR);
-    }
-
-    public cancelActiveRequest(downloadRequestId: string) {
-        this.cancellationRequests.add(downloadRequestId);
-        if (!this.activeRequestMap.hasOwnProperty(downloadRequestId)) {
-            return;
-        }
-
-        const { cancel } = this.activeRequestMap[downloadRequestId];
-        cancel();
-        delete this.activeRequestMap[downloadRequestId];
-    }
-
-    private download(options: DownloadOptions): Promise<DownloadResult> {
+    private async downloadOverHttp(options: DownloadOptions): Promise<DownloadResult> {
         const {
             downloadRequestId,
             encoding,
-            outFilePath,
             postData,
+            outFilePath,
             requestOptions,
             url,
             writeStreamOptions,
         } = options;
+        // let outFilePath = options.outFilePath;
+        // if (outFilePath === undefined) {
+        //     const defaultDirectory = await this.getDefaultDownloadDirectory();
+        //     outFilePath = path.join(defaultDirectory, path.basename(url));
+        // }
```

Comment:
still needed?

Reply:
oof nope removed in https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/106/commits/31d33b29fcb89c5faa32e62aab5f0637bfac79cb